index.rst

```diff
@@ -0,0 +1,20 @@
+.. tensorlayer documentation master file, created by
```

Comment:
do you mean `docs/index.rst` ??

Reply:
I have already removed index.rst and confirm.py